general/forall.hpp

```diff
@@ -91,7 +91,8 @@ using RAJA::statement::Segs;
 template <const int BLOCKS = MFEM_CUDA_BLOCKS, typename DBODY>
 void RajaCudaWrap1D(const int N, DBODY &&d_body)
 {
-   RAJA::forall<RAJA::cuda_exec<BLOCKS>>(RAJA::RangeSegment(0,N),d_body);
+   //true denotes asynchronous kernel
+   RAJA::forall<RAJA::cuda_exec<BLOCKS,true>>(RAJA::RangeSegment(0,N),d_body);
```

Comment:
@camierjs , kernels should be asynchronous right?

Reply:
The GPU kernels should be on the same stream I think, if this is the meaning of your question.

Reply:
Asynchronous yes, not sure to see the stream requirement.